src/main/java/org/infernalstudios/miningmaster/items/GemItem.java

```diff
@@ -15,14 +16,21 @@ public GemItem() {
         super(new Item.Properties());
     }
 
-
-
     @Override
     public void appendHoverText(ItemStack stack, @Nullable Level worldIn, List<Component> tooltip, TooltipFlag flagIn) {
         if (Screen.hasShiftDown()) {
-            tooltip.add(Component.literal("\u00A7dCombine with an item in a smithing table to enchant!"));
+            String tooltipText = this.getTooltipText().getString();
+            for (String line : tooltipText.split("\n")) {
+                tooltip.add(Component.literal(line));
+            }
```

Comment:
Note that this will not preserve color codes at the beginning of the string in the lang json. You should make sure to communicate this somehow.